extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/SmallRyeReactiveMessagingProcessor.java

```diff
@@ -89,6 +93,17 @@ public void validate(ValidationContext validationContext) {
                                 LOGGER.debugf("Found mediator business method %s declared on %s", method, bean);
                             }
                         }
+
+                        for (FieldInfo field : bean.getTarget().get().asClass().fields()) {
+                            if (annotationStore.hasAnnotation(field, NAME_STREAM)) {
+                                if (field.type().name().equals(
+                                        DotName.createSimple(Emitter.class.getName()))) {
```

Comment:
We should create a constant for `Emitter` dot name.

Reply:
Done